bootstrap.ps1

```diff
@@ -9,7 +9,10 @@ param(
     $NuGetVersion = $Env:NUGET_VERSION,
 
     [string]
-    $VsixVersion = $Env:VSIX_VERSION
+    $VsCodeVsixVersion = $Env:VSCODEVSIX_VERSION,
```

Comment:
VS Code has a hard requirement that its VSIX packages are SemVer 2.0 compliant. Given that, we possibly may not want a separate parameter for VS Code VSIX parameters.

Reply:
Thanks, you're right, this is redundant.